cincinnati/src/lib.rs

```diff
@@ -756,10 +756,11 @@ mod tests {
 
         let nodes: Vec<daggy::NodeIndex> = (start..(start + count))
             .map(|i| {
-                let metadata = metadata.remove(&i).unwrap_or_default();
+                let mut metadata = metadata.remove(&i).unwrap_or_default();
+                let suffix = metadata.remove("version_suffix").unwrap_or_default();
 
                 let release = Release::Concrete(ConcreteRelease {
-                    version: format!("{}.0.0", i),
+                    version: format!("{}.0.0{}", i, suffix),
```

Comment:
All of our exported graphs are going to be single-arch.  Do we need arch-specific names?  Or can we recycle "4.4.0" (say) to mean "the amd64 4.4.0" when the requested arch is amd64 and "the s390x 4.4.0" when the requested arch is s390x?

Reply:
This method is used for generating a test graph with arch suffixes, which are (will be) used in production in the processing chain and are only removed by the final arch filter plugin. Eventually we might ship them on a custom endpoint, or on requests which specifically ask for it, to enable multi-arch clusters.

Reply:
> Eventually we might ship them on a custom endpoint, or on requests which specifically ask for it, to enable multi-arch clusters.

Heterogenous clusters are going to need to use manifest-lists, at which point Cincinnati can go back to not caring about cluster architecture as far as nodes are concerned.  We might still care about cluster architecture as far as edge-stability is concerned.

Reply:
Gotcha, thanks for the outlook ;-) Can we conclude that we'll change now for the internal implementation, and we might change it back later when manifest-lists are generally available?